airflow/models/dag.py

```diff
@@ -1873,7 +1887,10 @@ def sync_to_db(self, session=NEW_SESSION):
 
         :return: None
         """
-        self.bulk_write_to_db([self], session=session)
+        # TODO: AIP-66 should this be in the model?
```

Comment:
you mean like, on the dag object, so we don't have to pass it along everywhere? yes, i think so

Reply:
We can look at this later 👍.